Mathlib/Data/Fin/VecNotation.lean

```diff
@@ -11,6 +11,8 @@ Authors: Anne Baanen
 import Mathlib.Data.Fin.Tuple.Basic
 import Mathlib.Data.List.Range
 import Mathlib.GroupTheory.GroupAction.Pi
+import Mathlib.Tactic.ToExpr
+import Qq
```

Comment:
This import seemed to fix some proofs in `Geometry/Manifold/ChartedSpace.lean`; any idea why?

Reply:
It's the `attribute [refl] Setoid.refl` in `Mathlib.Init.Align` that now gets imported